### ISSUE TITLE
Version 2.0: Class 'GravityView_Extension' not found

### ISSUE THREAD

**Issue by zackkatz** [gravityview/GravityView#973]
`GravityView_Plugin::include_extension_framework()` now just logs a notice, which would work okay if we checked to make sure that the extension was found before relying on it.

```
if( !class_exists( 'GravityView_Extension' ) ) {

		if( class_exists('GravityView_Plugin') && is_callable(array('GravityView_Plugin', 'include_extension_framework')) ) {
			GravityView_Plugin::include_extension_framework();
		} else {
			// We prefer to use the one bundled with GravityView, but if it doesn't exist, go here.
			include_once plugin_dir_path( __FILE__ ) . 'lib/class-gravityview-extension.php';
		}
	}

	class GravityView_Advanced_Filtering extends GravityView_Extension {
[...]
```

Any thoughts on how to handle this?

![Class 'GravityView_Extension' not found](http://share.gravityview.co/F07h0z+)

**Comment by soulseekah** [comment]
Why is it loading so early? Ideally it has to load on `gravityview/loaded` and not earlier, `init` is fine as well.

**Comment by soulseekah** [comment]
This only happens if Gravity Forms is deactivated.

**Comment by soulseekah** [comment]
Should be fixed in d00ebd9440241e86a08baa1e1acee630a4428081
